core/arch/arm/plat-imx/conf.mk

```diff
@@ -1,11 +1,20 @@
 PLATFORM_FLAVOR ?= mx6ulevk
 
-ifeq ($(PLATFORM_FLAVOR),mx6ulevk)
-arm32-platform-cpuarch		:= cortex-a7
+# Get SoC associated with the PLATFORM_FLAVOR
+ifeq ($(PLATFORM_FLAVOR),$(filter $(PLATFORM_FLAVOR),mx6ulevk))
+$(call force,CFG_MX6UL,y)
 endif
-ifeq ($(PLATFORM_FLAVOR),$(filter $(PLATFORM_FLAVOR),mx6qsabrelite mx6qsabresd mx6dlsabresd))
-arm32-platform-cpuarch		:= cortex-a9
+
+ifeq ($(PLATFORM_FLAVOR),$(filter $(PLATFORM_FLAVOR),mx6qsabrelite mx6qsabresd))
```

Comment:
@etienne-lms I'm not arguing against giving all the supported platforms explicitly and printing an error message if it's not recognized.
I'm complaining about the syntax : "if the value of A is x or y" is better written as `ifneq (,$(filter $(A),x y))` than `ifeq ($(A),$(filter $(A),x, y))` because (1) it's shorter and (2) it handles the case where A is undefined or empty properly. Or...?